jenkins/opensearch/Jenkinsfile

```diff
@@ -103,6 +104,7 @@ pipeline {
                     steps {
                         script {
                             build()
+                            test()
```

Comment:
Food for thought, adding the testing immediately after the build step reuses the same docker host (OK!).  We could break the testing into its own stage so we can make the message more granular, then you can create a `post() { failure }` section to deal with tests failing, but the build not failing.